server/service/IndividualServicesService.js

```diff
@@ -152,67 +158,70 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
-          update.isProtocolUpdated = await tcpClientInterface.setRemoteProtocolAsync(newReleaseTcpClientUuid, applicationProtocol);
-        }
-        if(applicationAddress != currentNewReleaseRemoteAddress) {
-          update.isAddressUpdated = await tcpClientInterface.setRemoteAddressAsync(newReleaseTcpClientUuid, applicationAddress);
+        if (currentNewReleaseNumber != releaseNumber) {
+          update.isReleaseUpdated = await httpClientInterface.setReleaseNumberAsync(newReleaseHttpClientLtpUuid, releaseNumber);
         }
-        if(applicationPort != currentNewReleaseRemotePort) {
-          update.isPortUpdated = await tcpClientInterface.setRemotePortAsync(newReleaseTcpClientUuid, applicationPort);
+        if (applicationName != currentNewReleaseApplicationName) {
+          update.isApplicationNameUpdated = await httpClientInterface.setApplicationNameAsync(newReleaseHttpClientLtpUuid, applicationName);
         }
 
-        let isdataTransferRequired = true;
-        let serverAddress = await tcpServerInterface.getLocalAddressOfTheProtocol(applicationProtocol);
-         let serverPort = await tcpServerInterface.getLocalPortOfTheProtocol(applicationProtocol);
-          if ( applicationAddress === serverAddress && applicationPort === serverPort) {         
-               isdataTransferRequired = false;
-                 }
-
-        if (update.isProtocolUpdated || update.isAddressUpdated || update.isPortUpdated) {
+        if (update.isReleaseUpdated || update.isApplicationNameUpdated) {
           let configurationStatus = new ConfigurationStatus(
-            newReleaseTcpClientUuid,
+            newReleaseHttpClientLtpUuid,
             undefined,
             true);
-            logicalTerminationPointConfigurationStatus.tcpClientConfigurationStatusList = [configurationStatus]; } 
-                  if (logicalTerminationPointConfigurationStatus != undefined) {
-
-          /****************************************************************************************
-           * Prepare attributes to automate forwarding-construct
-           ****************************************************************************************/
-          let forwardingAutomationInputList = await prepareForwardingAutomation.bequeathYourDataAndDie(
-            logicalTerminationPointConfigurationStatus
-          );
-          ForwardingAutomationService.automateForwardingConstructAsync(
-            operationServerName,
-            forwardingAutomationInputList,
-            user,
-            xCorrelator,
-            traceIndicator,
-            customerJourney
-          );
+          logicalTerminationPointConfigurationStatus.httpClientConfigurationStatus = configurationStatus;
+          // ALT should know about this change
+
+          let newReleaseTcpClientUuidList = await logicalTerminationPoint.getServerLtpListAsync(newReleaseHttpClientLtpUuid);
+          let newReleaseTcpClientUuid = newReleaseTcpClientUuidList[0];
```

Comment:
According to the issue , these changes are not required . Please remove the unwanted changes